samples/word/99-preview-apis/manage-list-styles.yaml

```diff
@@ -0,0 +1,112 @@
+id: word-lists-manage-styles
+name: Manage list styles
+description: This sample shows how to get the list styles in the current document.
+author: xiruatms
+host: WORD
+api_set:
+    WordApi: '1.9'
+script:
+    content: |
+        $("#count").on("click", () => tryCatch(getCount));
+        $("#properties").on("click", () => tryCatch(getProperties));
+
+        async function getCount() {
+          // Gets the available list styles stored with the document.
+          await Word.run(async (context) => {
+            const styles: Word.StyleCollection = context.document.getStyles();
+            const count = styles.getCount();
+            styles.load();
```

Comment:
Here and elsewhere, let's avoid loading more properties than necessary to show best practices.

Reply:
I'm loading everything for now so users can see all the properties in the log. Maybe I'll add a code comment to that effect, which will explain why we're not strictly following best practice.